runtime/gc_modron_startup/mgcalloc.cpp

```diff
@@ -114,9 +114,13 @@ J9AllocateObjectNoGC(J9VMThread *vmThread, J9Class *clazz, uintptr_t allocateFla
 			if (NULL != objectPtr) {
 				uintptr_t allocatedBytes = env->getExtensions()->objectModel.getConsumedSizeInBytesWithHeader(objectPtr);
 				Assert_MM_true(allocatedBytes == mixedOAM.getAllocateDescription()->getContiguousBytes());
-				if (J9_ARE_ANY_BITS_SET(J9CLASS_EXTENDED_FLAGS(clazz), J9ClassReservableLockWordInit)) {
-					j9objectmonitor_t *lockEA = J9OBJECT_MONITOR_EA(vmThread, objectPtr);
-					J9_STORE_LOCKWORD(vmThread, lockEA, OBJECT_HEADER_LOCK_RESERVED);
+				if (LN_HAS_LOCKWORD(vmThread, objectPtr)) {
+					j9objectmonitor_t initialLockword = VM_ObjectMonitor::getInitialLockword(vmThread->javaVM, clazz);
+
```

Comment:
Would you please remove this empty line?

Reply:
I'll remove it.